src/www/.storybook/localizer_factory.js

```diff
@@ -0,0 +1,20 @@
+/* tslint:disable */
+/*
+  Copyright 2022 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import * as locales from "../messages";
+
+export const localizerFactory = (locale = "English") => messageId => locales[locale][messageId];
```

Comment:
Wait, is this just outputting the message template? What about parameters?

This is how we do it in the Gallery app. We could do the same here: https://github.com/Jigsaw-Code/outline-server/blob/84e83c87197b1e30288e24926022ba237834d0a1/src/server_manager/web_app/gallery_app/main.ts#L39

Reply:
Resolving in favor of new PR.